### ISSUE TITLE
Problems with generating prisma client using bun

### ISSUE THREAD

**Issue by Elijjahh** [oven-sh/bun#14962]
## What version of Bun is running?

1.1.33

## What platform is your computer?

Macos 15.1

## What is the expected behavior?
I should get generated client.

## What do you see instead?
When I install @prisma/nuxt and start at dev mode everything works until "Generating Prisma client...". At this moment it just freezes and nothing happens

## Steps to reproduce:

```
# creating nuxt project
bunx nuxi@latest init <my-app>

bun install @prisma/nuxt
```

Add "@prisma/nuxt" to nuxt modules and run:
```
bun run dev
```

## Additional information
There are some experiments that I've done. I've noticed that it works when there is no bun.lockb. So I tried installing packages and start dev mode with deleted bun.lockb and it worked.

Also if I add bunfig.toml:
```
[install.lockfile]

# whether to save the lockfile to disk
save = false
```
then it also works.

Also after successful first time (using no bun.lockdb or bunfig.toml)  it works anyway (with/without bun.lockdb). I am new to bun and prisma so I don't know the exact reason. I guess that "@prisma/nuxt" installs "@prisma/client" at first time and that breaks everything and at second time it is already installed so everything works. And it's very strange that bun.lockdb somehow affects runtime

### Screenshots
Generation freezes. You can see that it started at 22:03 and I made a screenshot at 22:06:

![изображение](https://github.com/user-attachments/assets/54ce2d39-49dc-4f50-a38c-ade311f47815)

Generation works good. Removed @prisma/nuxt from modules so it doesn't run at nuxt prepare. Then removed bun.lockdb and ran bun dev:

<img width="1440" alt="Снимок экрана 2024-11-01 в 22 12 12" src="https://github.com/user-attachments/assets/485dfd34-89d0-40a1-bd98-ea475f891d76">
<img width="1440" alt="Снимок экрана 2024-11-01 в 22 13 33" src="https://github.com/user-attachments/assets/6512a1d7-eeac-4c46-b697-3f464afc0c76">

Generation works. Added bunfig.toml

<img width="1440" alt="Снимок экрана 2024-11-01 в 22 17 11" src="https://github.com/user-attachments/assets/ce323ba9-e090-4167-8c68-f3c96c1f2be8">

**Comment by realspinelle** [comment]
in case your searching for a fix downgrade ur bun to 1.1.29

**Comment by Elijjahh** [comment]
For anyone with the same problem upgrading to 1.1.38 fixed it! I didn't try to test with versions between 1.1.33 and 1.1.38 so may be bug was fixed earlier. For me it doesnt matter) So I am closing this issue. 

Thanks @Jarred-Sumner for your care
